include/modem/lte_lc.h

```diff
@@ -651,6 +651,47 @@ enum lte_lc_reduced_mobility_mode {
 	LTE_LC_REDUCED_MOBILITY_DISABLED = 2,
 };
 
+/**
+ * UICC power saving mode.
```

Comment:
We should add this text somewhere. Maybe it's best to add it here, so don't need to have it in both functions.
```suggestion
 * UICC power saving mode.
 *
 * Affects only UICC deactivation and suspension during eDRX and idle state. PSM functionality
 * is not affected.
```

Reply:
I'd reword your suggestion to avoid confusion that this affects UICC deactivation and suspension functionality. I assume it's more about those being used to achieve power savings in eDRX and idle states.

Reply:
> I'd reword your suggestion to avoid confusion that this affects UICC deactivation and suspension functionality. I assume it's more about those being used to achieve power savings in eDRX and idle states.

What would your rewrite it to? I don't understand the nuances of how it's different from @tokangas version.